### PR TITLE
Changes in the Swift Lint.

### PR DESCRIPTION
**Pull request by shtomar-adb**
## Description

Changes in the Swift Lint. Use podspec for installing SwiftLint.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by rymorale** [comment]
these changes look good to me although the latest commit appears to have introduced some lint issues.

**Review by rymorale — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -33,28 +42,22 @@ test:
...
-
-lint-autocorrect:
-	swiftlint autocorrect
+install-swiftformat:
+	(brew install swiftformat) 
 
-checkFormat:
-	swiftformat . --lint --swiftversion 5.2
+check-format:
+	(swiftformat --lint $(PROJECT_NAME)/Sources --swiftversion 5.2)
```
Comment (rymorale) [inline Makefile:57]: i believe we want to use 5.1 to be in sync with other AEPSDK extensions.

**Review by rymorale — APPROVED** [review]
looks good to me with just a minor update needed for the swiftformat swift version

**Review by swarna04 — CHANGES_REQUESTED** [review]

**Review by swarna04 — APPROVED** [review]
👍

**Pull request merged**